### ISSUE TITLE
[Accessibility] [Supporting the platform -Intro]: user is not able to access the scroll bar in code part when the page is zoomed to 200%. [Sev3][WCAG1.4.4]

### ISSUE THREAD

**Issue by Laxmidurga** [microsoft/tsdoc.org-website#32]
**Test Environment:**
OS Build: Windows 11
Version: 21H2 (OS Build 22000.318)
Browser: Edge
Version 96.0.1054.43 (Official build) (64-bit)
URL: https://tsdoc.org/
Screen reader: Narrator

**Prerequisites:** Set the Display resolution to 1280*768 and browser zoom to 200%

**Repro Steps:**
1.	Open the above URL and login with valid credentials.
2.	TSDoc page will get displayed.
3.	Now, navigate to the ‘Intro’ link and activate it.
4.	Now set the display resolution to 1280*768 and browser zoom to 200% and check whether the user is able to access the scroll bar in code part when the page is zoomed to 200%. 

**Actual:**
user is not able to access the scroll bar in code part when the page is zoomed to 200%.

**Expected:**
user should be able to access the scroll bar in code part when the page is zoomed to 200%.

**User Impact:**
People with mild visual disabilities must be able to resize text up to 200 percent without losing access to content or losing functionality. 

**Note:**
Issue is observed in Tags, submitting a PR, @returns pages for both 200% and 400%.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/resize-text

https://user-images.githubusercontent.com/88436727/150644248-368ffc58-7536-45f9-bfee-2432414870c3.mp4

**Comment by octogonz** [comment]
This issue has been closed because this Jekyll project has been retired. The documentation content has been migrated to Docusaurus. 

If the problem has not been addressed, please file a new issue under this GitHub project instead:  

https://github.com/microsoft/rushstack-websites
